integration_tests/sdk/aqueduct_tests/external_compute_test.py

```diff
@@ -0,0 +1,57 @@
+import platform
+
+import pytest
+from aqueduct.error import AqueductError
+
+from aqueduct import check, global_config, op
+
+from ..shared.data_objects import DataObject
+from ..shared.flow_helpers import publish_flow_test
+from .extract import extract
+
+
+@pytest.mark.enable_only_for_external_compute()
+def test_flow_with_multiple_compute_using_op_spec(client, flow_name, data_integration, engine):
+    """Runs a workflow both Aqueduct and a third-party compute engine."""
+    table_artifact = extract(data_integration, DataObject.SENTIMENT)
+
+    @op
+    def noop(input):
+        return input
+
+    @op(engine=engine, requirements=[])
+    def noop_on_third_party(input):
+        return input
+
+    # Only `noop_on_third_party` is run on outside compute.
+    output = noop_on_third_party(noop(table_artifact))
+    publish_flow_test(client, output, name=flow_name(), engine=None)
```

Comment:
@kenxu95 what are we actually testing here? Are we just testing to make sure nothing errored? Also why do we set `engine=None`?

Reply:
Yeah we're just testing that a flow with multiple external computes does not error. I can add a line for checking the artifact values too.

We set engine to None so that `noop()` is executed by Aqueduct, but `noop_on_third_party()` is executed on the external compute engine.